packages/next-auth/src/providers/email.ts

```diff
@@ -1,13 +1,13 @@
 import { createTransport } from "nodemailer"
 
 import type { CommonProviderOptions } from "."
-import type { Options as SMTPConnectionOptions } from "nodemailer/lib/smtp-connection"
+import type { Options as SMTPTransportOptions } from "nodemailer/lib/smtp-transport"
```

Comment:
Does this type exist? I can't see it on **DefinitivelyTyped**:
https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/nodemailer/lib

Are we use both types have precisely the same options? We don't want this to be a breaking change 💅🏽

Reply:
this type is here.
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/nodemailer/lib/smtp-transport.d.ts#L42

Options named SMTPTransportOptions extends Options named SMTPConnectionOptions.

Reply:
I see, so this type [`SMTPTransportOptions`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/nodemailer/lib/smtp-transport.d.ts#L42-L46) extends [`SMTPConnectionOptions`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/nodemailer/lib/smtp-connection/index.d.ts#L131-L172) with few more things 🤔 

In that case it's ok for me 👍🏽